### PR TITLE
update action scripts

### PR DESCRIPTION
**Pull request by raymondtfr**

### CODE REVIEW

File: data/actions/scripts/other/music.lua [hunk 1]
```diff
@@ -28,42 +28,47 @@ local instruments = {
 	[3953] = {effect = CONST_ME_SOUND_RED}, -- war drum
 	[3957] = {effect = CONST_ME_SOUND_YELLOW, item = {id = 2681, count = 10}, chance = 80, remove = true}, -- cornucopia
 	[5786] = {effects = {failure = CONST_ME_SOUND_RED, success = CONST_ME_SOUND_YELLOW}, monster = "war wolf", chance = 60, remove = true}, -- wooden whistle
-	[6572] = {effect = CONST_ME_SOUND_GREEN, text = "TOOOOOOT", transformId = 13578}, -- party trumpet
-	[6573] = {effect = CONST_ME_SOUND_GREEN, text = "TOOOOOOT", transformId = 13578}, -- party trumpet
+	[6572] = {effect = CONST_ME_SOUND_GREEN, text = "TOOOOOOT", transform = {id = 13578}}, -- party trumpet
+	[6573] = {effect = CONST_ME_SOUND_GREEN, text = "TOOOOOOT", transform = {id = 13578}}, -- party trumpet
```
Comment (WibbenZ) [inline data/actions/scripts/other/music.lua]: Why do you keep making the item id in another table when all the parent table has is that value?
Just name it transformId or w/e but you don't have to make transform a table, unless you add support to actually transform into more than one item.

Comment (raymondtfr) [inline data/actions/scripts/other/music.lua]: So that it can be easily customizated and also adds more maintainability to the code.

Comment (ranisalt) [inline data/actions/scripts/other/music.lua]: Make the simplest code that works. If new features are needed, they are to be added later, not now.

File: data/actions/scripts/other/constructionkits.lua [hunk 2]
```diff
@@ -1,4 +1,4 @@
-local constructionKits = {
+local kits = {
```
Comment (ranisalt) [inline data/actions/scripts/other/constructionkits.lua]: No need to rename the variable...

File: data/actions/scripts/other/constructionkits.lua [hunk 3]
```diff
@@ -8,24 +8,23 @@ local constructionKits = {
...
+	if not Tile(fromPosition):getHouse() then
 		player:sendTextMessage(MESSAGE_STATUS_SMALL, "You may construct this only inside a house.")
-	else
-		item:transform(kit)
-		fromPosition:sendMagicEffect(CONST_ME_POFF)
+		return true
 	end
+
+	item:transform(kits[item:getId()])
+	fromPosition:sendMagicEffect(CONST_ME_POFF)
```
Comment (ranisalt) [inline data/actions/scripts/other/constructionkits.lua]: I didn't understand why you changed the `if/elseif/else` structure.

File: data/actions/scripts/other/decayto.lua [hunk 4]
```diff
@@ -1,16 +1,11 @@
-local decayItems = {
+local items = {
```
Comment (ranisalt) [inline data/actions/scripts/other/decayto.lua]: No need to rename this one too.

File: data/actions/scripts/other/constructionkits.lua (hunk 3 above)
Comment (raymondtfr) [inline data/actions/scripts/other/constructionkits.lua]: I find it more clear not to use them, but I can revert it if there is no need.

**Comment by raymondtfr** [comment]
Later I'll send a new PR with these changes (much better than dealing with conflicting files 😄 . I'll be closing this for now.

**Pull request closed without merging**